Documentation/Signoff/Wireless Power Transfer.md

```diff
@@ -0,0 +1,244 @@
+# Wireless Power Transfer
+
+## Function 
+
+The purpose of this subsystem is to demonstrate wireless charging, energy, and power concepts through user interaction. The voltage will be adjustable by the user to control different charge rates, allowing them to analyze different drive times of the car depending on the power supply and time of charge. This will allow the user to learn how power and energy behave through the use of wireless charging. 
+
+## Constraints
+
+|Description    |Constraint                                                                                                            |Source                      |
+|---------------|----------------------------------------------------------------------------------------------------------------------|----------------------------|
+|Standard       |The system shall operate at less than 50 V to ensure safety in an educational setting.                                |OSHA 1926.403(i)(2)(i)      |
```

Comment:
I see nothing about 50 volts in that standard.

Reply:
I'm confused on what you mean. I know that osha 1926.403(i)(2)(i) states the following: "Except as required or permitted elsewhere in this subpart, live parts of electric equipment operating at 50 volts or more shall be guarded against accidental contact by cabinets or other forms of enclosures, or by any of the following means:". I have it references but maybe I'm misunderstanding what you're saying.

Reply:
You must have the wrong OSHA standard number.